card.js

```diff
@@ -0,0 +1,37 @@
+const projects = [
+  {
+    id: 1,
+    title: 'Tonic',
+    image: 'images/portfolio-1.svg',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['html', 'css', 'javaScript'],
+    source: 'https://github.com/josuerushanika/Josue-portfolio',
+  },
+  {
+    id: 2,
```

Comment:
- [ ] kindly add the height of your mobile popup to avoid the background view and also follow the Figma design
- [ ] please kindly ensure that your `seesource `and `seelive `buttons are of same size.

![Screenshot (718)_LI](https://user-images.githubusercontent.com/65298048/210845208-26afc532-cc71-4ccf-9fc7-8f26c514519c.jpg)

Reply:
Thanks so much it's fixed